packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,50 @@
+/* globals JitsiMeetExternalAPI */
+
+Template.videoFlexTab.helpers({
+
+});
+
+Template.videoFlexTab.onCreated(function() {
+	let rid = this.data && this.data.rid;
+	this.timeout = null;
+	this.autorun(() => {
+		if (RocketChat.settings.get('Jitsi_Enabled')) {
+			if (RocketChat.TabBar.getTemplate() === 'videoFlexTab') {
+				if (RocketChat.TabBar.isFlexOpen()) {
+					Meteor.call('jitsi:connect', rid);
+				} else {
+					Meteor.call('jitsi:disconnect', rid);
+				}
+			}
+		}
+	});
+
+	// Opening a PR so we can do this via the https://meet.jit.si/external_api.js
+	$.getScript('https://cdn.rawgit.com/geekgonecrazy/jitsi-meet/master/external_api.js')
+		.done(function(script, textStatus) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["textStatus" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2707254899/issues/source?bid=3199634&fileBranchId=3342079#l24)